mods/ts/rules/gdi-infantry.yaml

```diff
@@ -91,12 +92,12 @@ GHOST:
 		Cost: 1750
 	Tooltip:
 		Name: Ghost Stalker
-		Description: Elite commando infantry, armed with \ndual pistols and C4.\n  Strong vs Infantry, Buildings\n  Weak vs Vehicles\n  Special Ability: Destroy Building with C4\n\nMaximum 1 can be trained
-		BuildLimit: 1
+		Description: Elite commando infantry, armed with \ndual pistols and C4.\n  Strong vs Infantry, Buildings\n  Weak vs Vehicles\n  Special Ability: Destroy Building with C4\nMaximum 1 can be trained
```

Comment:
You should fix the Tanya description here to be actually valid for the Ghost stalker, who doesn't have dual pistols.